edrumulus_parameters.cpp

```diff
@@ -57,6 +57,99 @@ void Edrumulus::Pad::apply_preset_pad_settings()
 
   switch ( pad_settings.pad_type )
   {
+
+    case LEHHS12C: // dual trigger
+      pad_settings.is_rim_switch           = true;
+      pad_settings.scan_time_ms            = 4.0f;
+      pad_settings.decay_fact_db           = 5.0f;
+      pad_settings.decay_len2_ms           = 600.0f;
+      pad_settings.decay_grad_fact2        = 100.0f;
+      pad_settings.rim_shot_is_used        = true;
+      pad_settings.rim_shot_treshold       = 25;
+      pad_settings.rim_shot_boost          = 0;
+      pad_settings.velocity_threshold      = 18;
+      pad_settings.velocity_sensitivity    = 6;
+      break;
+
+    case LEHHS12C_CTRL: // 0-35k poti
+      pad_settings.is_control              = true;
+      pad_settings.pos_threshold           = 6;
+      pad_settings.pos_sensitivity         = 23;
+      pad_settings.velocity_threshold      = 0;
+      pad_settings.velocity_sensitivity    = 22;
+      break;
+
+  /* MPS 750X:
```

Comment:
This is not the right place to give the users hints for the schematics. I would recommend to move your text in the manual at https://github.com/corrados/edrumulus/blob/main/doc/manual.md#analog-front-end-circuit

BTW, most parameters like scan time, pre scan time, decay len, etc. do not depend at all from the chosen resistors. Basically, it is the velocity sensitivity which is mainly influenced by the selection of the resistors. But since most users will change this parameter to their playing style, it is not that much critical.